core/src/main/java/org/wildfly/channel/ChannelMapper.java

```diff
@@ -67,6 +66,7 @@ public class ChannelMapper {
 
     static {
         SCHEMAS.put(SCHEMA_VERSION_1_0_0, SCHEMA_FACTORY.getSchema(ChannelMapper.class.getClassLoader().getResourceAsStream(SCHEMA_1_0_0_FILE)));
+        SCHEMAS.put(SCHEMA_VERSION_2_0_0, SCHEMA_FACTORY.getSchema(ChannelMapper.class.getClassLoader().getResourceAsStream(SCHEMA_2_0_0_FILE)));
```

Comment:
I think the mapper as it is here cannot read the old version of the channel (v1)? It ends up with null manifest and manifest coordinate. It can be OK to leave it this way, there are no real channels out there yet. But we have plenty of v1 channels in prospero tests etc which we will not be able to read properly, maybe other tooling has them too. So either we can implement mapper ability to read v1 channels or we have to transform all our channels to v2. WDYT?

Reply:
In general, yes we'd need some sort of transformer. I'm not sure if it's worth doing in this case as essentially channel v1 becomes manifest v1 rather then channel v2.

Reply:
OK, I will convert the channels.